Resources/Prototypes/_Goobstation/Loadouts/loadout_groups.yml

```
@@ -50,6 +50,14 @@
   - BlueshieldOfficerHeadBeretNavy
   - BlueshieldOfficerHeadBeretBlack
 
+- type: loadoutGroup
+  id: BlueshieldOfficerBackpack
+  name: loadout-group-bso-backpack
+  loadouts:
+  - BlueshieldOfficerBackpack
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)